packages/fluentui/react-northstar/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -8,6 +8,7 @@ export const popupContentStyles: ComponentSlotStylesPrepared<PopupContentStylesP
   root: ({ props: p, variables: v }): ICSSInJSStyle => ({
     display: 'block',
     zIndex: v.zIndex,
+    position: 'fixed',
```

Comment:
@layershifter This is needed only for the autoFocus input example, Datepicker works fine with added modifiers